priya/javascript/Assignment11/JsAssign11Q1.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Roll number and Marks</title>
+  </head>
+  <body>
+    Enter number: <input type="text" id="pic">
+    <button onclick="show_roll()">show</button>
+    <p id="demo"></p>
+    <script>
+      function show_roll() {
+        var p;
+        p = parseInt(document.getElementById("pic").value);
+        document.getElementById("demo").innerHTML = `roll no :${p} scrored ${sorted_mark[roll_no.indexOf(p)]} marks`;
+      }
+      //take the roll no from user
+      var roll_no = [];
+      for (var index = 0; index <= 9; index++) {
+        roll_no[index] = parseInt(prompt("Enter a  roll number"));
+      }
+      document.write("Roll numbers are : <br> <br>" + roll_no + "<br>");
```

Comment:
Comment From Here to
```suggestion
     /* document.write("Roll numbers are : <br> <br>" + roll_no + "<br>");
```